1-js/02-first-steps/09-comparison/article.md

```diff
@@ -1,214 +1,191 @@
-# Comparisons
+# तुलना
 
-We know many comparison operators from maths.
+हम गणित से कई तुलना ऑपरेटरों को जानते हैं।
```

Comment:
**Operators** here refers to the **notations**, which should be translated as **प्रतीकों**. [Refer to this.](https://hi.wikipedia.org/wiki/%E0%A4%97%E0%A4%A3%E0%A4%BF%E0%A4%A4%E0%A5%80%E0%A4%AF_%E0%A4%AA%E0%A5%8D%E0%A4%B0%E0%A4%A4%E0%A5%80%E0%A4%95%E0%A5%8B%E0%A4%82_%E0%A4%95%E0%A5%80_%E0%A4%B8%E0%A4%BE%E0%A4%B0%E0%A4%A3%E0%A5%80)

Reply:
/done